content/include/mock_assert_example.md

```diff
@@ -0,0 +1,33 @@
+    // ...
+    t.Run("nominal case", func(t *testing.T) {
+        t.Parallel()
+
+        mock := mocks.BaselineTransport(t)
```

Comment:
I get the overall idea but don't get why the mock isn't used in the example besides its definition.